lib/seahorn/BvOpSem2ExtraWideMemMgr.cc

```diff
@@ -26,6 +26,7 @@ ExtraWideMemManager<T>::ExtraWideMemManager(Bv2OpSem &sem, Bv2OpSemContext &ctx,
       m_offset(sem, ctx, ptrSz, ptrSz, useLambdas, true),
       m_size(sem, ctx, ptrSz, g_slotByteWidth, useLambdas, true),
       m_uninit_size(m_ctx.alu().ui(g_uninit_size, g_slotBitWidth)),
+      m_freshPtrName(mkTerm<std::string>("sea.ptr", m_efac)), m_id(0),
```

Comment:
why should this functionality be duplicated from RawMemManager and not simply reused from there. All the wide mem managers already depend on other, simpler, mem managers, and the raw one provides names to allocator.

If this is duplicated, at least the names should be different, i.e., not `sea.ptr`, to simplify debugging later on. But I really think it should be properly refactored.

Reply:
Good point! fixed. Will submit after tests complete.